src/hooks/useFetchUser.ts

```diff
@@ -0,0 +1,63 @@
+import { useQuery } from '@tanstack/react-query';
+import { useWallet } from 'src/providers';
+import type { FeatureCardData } from 'src/types';
+
+export interface UseFetchUserProps {
+  featureCards: FeatureCardData[] | undefined;
+  isSuccess: boolean;
+  isConnected: boolean;
+}
+
+const STRAPI_CONTENT_TYPE = 'jumper-users';
+export const useFetchUser = (): UseFetchUserProps => {
+  const { account } = useWallet();
+
+  const apiBaseUrl =
+    import.meta.env.VITE_STRAPI_DEVELOP === 'true'
+      ? import.meta.env.VITE_LOCAL_STRAPI_URL
+      : import.meta.env.VITE_STRAPI_URL;
+  const apiUrl = new URL(`${apiBaseUrl}/${STRAPI_CONTENT_TYPE}`);
+  apiUrl.searchParams.set('populate[0]', 'feature_cards');
+  apiUrl.searchParams.set(
+    'populate[feature_cards][populate][0]',
+    'BackgroundImageLight',
+  );
+  apiUrl.searchParams.set(
+    'populate[feature_cards][populate][1]',
+    'BackgroundImageDark',
+  );
+  if (account.address) {
+    apiUrl.searchParams.set(
+      'filters[EvmWalletAddress][$eqi]',
+      account?.address,
+    );
+  }
+  import.meta.env.MODE !== 'production' &&
+    apiUrl.searchParams.set('publicationState', 'preview');
+  const apiAccesToken =
+    import.meta.env.VITE_STRAPI_DEVELOP === 'true'
+      ? import.meta.env.VITE_LOCAL_STRAPI_API_TOKEN
+      : import.meta.env.VITE_STRAPI_API_TOKEN;
```

Comment:
We have configs for all environments so maybe there should be only one `VITE_STRAPI_API_TOKEN` variable with different values for different envs?